package.json

```diff
@@ -1,9 +1,9 @@
 {
   "name": "autocannon-ui",
-  "version": "1.0.0",
-  "private": true,
+  "version": "2.0.0",
```

Comment:
don't bump the version, this is done when we release it

Reply:
Yepp, reverted